src/main/java/com/yuriytkach/tracker/fundraiser/service/TrackService.java

```diff
@@ -180,6 +182,22 @@ private SlackResponse processCreateFundCommand(final Matcher matcher, final Stri
     }
   }
 
+  private SlackResponse processUpdateFundCommand(final Matcher matcher, final String user) {
```

Comment:
We need to check that the fund that is being updated is owned by the `user`. Take a look how we process `delete` command.

Reply:
Added

Reply:
Btw, what is the purpose to mark all variables as final otherwise codacy doesn't allow you to pass checking

Reply:
To enforce immutability